### ISSUE TITLE
E322 false positive

### ISSUE THREAD

**Issue by TotalVerb** [tonyhffong/Lint.jl#142]
```julia
julia> lintstr("""
       module X
           function x end
           export x
       end
       """)
1-element Array{Lint.LintMessage,1}:
 none:3 E322 x: exporting undefined symbol
```

**Comment by TotalVerb** [comment]
Also, E322 should be a warning. It can never lead to program failure, and it meets the definition of a warning quite well: Code that will run but is probably wrong.

**Comment by Michael-Klassen** [comment]
I agree. `Error` should only be used on things guaranteed to fail.

**Comment by tonyhffong** [comment]
I concur.
